org.jellyfin.JellyfinServer.Plugin.IntelComputeRuntime.yml

```diff
@@ -0,0 +1,137 @@
+            url: https://github.com/llvm/llvm-project/archive/refs/tags/llvmorg-11.1.0.tar.gz
+            sha256: 53a0719f3f4b0388013cfffd7b10c7d5682eece1929a9553c722348d1f866e79
+            dest: llvm-project
+          - type: git
+            url: https://github.com/intel/opencl-clang.git
+            commit: 363a5262d8c7cff3fb28f3bdb5d85c8d7e91c1bb
+            dest: llvm-project/llvm/projects/opencl-clang
+          - type: git
+            url: https://github.com/KhronosGroup/SPIRV-LLVM-Translator.git
+            commit: 4ef524240833abfeee1c5b9fff6b1bd53f4806b3
+            dest: llvm-project/llvm/projects/llvm-spirv
+          - type: patch
+            path: no_git_patching.patch
+          - type: shell
+            commands:
+              - cat llvm-project/llvm/projects/opencl-clang/patches/{clang,llvm}/*.patch | patch -Np1 -d llvm-project
+              - cat llvm-project/llvm/projects/opencl-clang/patches/spirv/*.patch | patch -Np1 -d llvm-project/llvm/projects/llvm-spirv
+      - name: intel-gmmlib
+        buildsystem: cmake-ninja
+        builddir: true
+        sources:
+          - type: archive
+            url: https://github.com/intel/gmmlib/archive/refs/tags/intel-gmmlib-22.3.0.tar.gz
+            sha256: c1f33e1519edfc527127baeb0436b783430dfd256c643130169a3a71dc86aff9
+      - name: level-zero
+        buildsystem: cmake-ninja
+        sources:
+          - type: archive
+            url: https://github.com/oneapi-src/level-zero/archive/refs/tags/v1.8.8.tar.gz
+            sha256: 3553ae8fa0d2d69c4210a8f3428bd6612bd8bb8a627faf52c3658a01851e66d2
+
+  - name: khronos-opencl-headers
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/KhronosGroup/OpenCL-Headers/archive/refs/tags/v2023.02.06.tar.gz
+        sha256: 464d1b04a5e185739065b2d86e4cebf02c154c416d63e6067a5060d7c053c79a
+        x-checker-data:
+          type: anitya
+          project-id: 223257
+          stable-only: true
+          url-template: https://github.com/KhronosGroup/OpenCL-Headers/archive/refs/tags/v$version.tar.gz
+
+  - name: ocl-icd
+    config-opts:
+      - --disable-update-database
+      - --enable-custom-vendordir=/app/extensions/OpenCL/vendors
+      - --enable-official-khronos-headers
+    sources:
+      - type: archive
+        url: https://github.com/OCL-dev/ocl-icd/archive/refs/tags/v2.3.1.tar.gz
+        sha256: a32b67c2d52ffbaf490be9fc18b46428ab807ab11eff7664d7ff75e06cfafd6d
+        x-checker-data:
+          type: anitya
+          project-id: 2525
+          stable-only: true
+          url-template: https://github.com/OCL-dev/ocl-icd/archive/refs/tags/v$version.tar.gz
+
+  - name: clinfo
+    buildsystem: simple
+    build-commands:
+      - make -j$FLATPAK_BUILDER_N_JOBS PREFIX=$FLATPAK_DEST/extensions install
+      - chmod 755 $FLATPAK_DEST/extensions/bin/clinfo
+      - install -D $FLATPAK_ID.metainfo.xml $FLATPAK_DEST/share/metainfo/$FLATPAK_ID.metainfo.xml
```

Comment:
this will not install where you want. `$FLATPAK_DEST` is set to what the prefix is.